include/solvers/navier_stokes_scratch_data.h

```diff
@@ -808,6 +854,12 @@ class NavierStokesScratchData
   // This is stored as a shared_ptr because it is only instantiated when needed
   std::shared_ptr<FEValues<dim>> fe_values_VOF;
 
+  bool                           gather_pfg;
+  bool                           gather_curvature;
```

Comment:
Are there occasion where we will want to gather pfg and NOT the curvature? otherwise why not always gather both and make things simpler for the reader/user

Reply:
I think it's more modular this way. Besides, we may need to gather PFG without curvature for other purposes. But I can change it if you think it's better to use the same function for both.

Reply:
Sorry I just catch that: why this `pgf` has not been changed to `filtered_phase_fraction_gradient` (or another name, see other comment)? Please check that the denomination has been changed everywhere, otherwise we will have two name for the same thing and that will be confusing